launch/planning_scene/planning_scene_visualization.launch

```diff
@@ -1,18 +1,14 @@
 <?xml version="1.0"?>
 <launch>
 
-  <arg name="world_config" default="ai_day" doc="objects arrangement, options: moelk_tables, cic_tables, truck_assembly, ai_day" />
-  <arg name="yaml_path_to_read" default="$(find tables_demo_bringup)/config/$(arg world_config)_planning_scene.yaml" />
-  <arg name="yaml_path_to_write" default="$(arg yaml_path_to_read)" />
+  <arg name="yaml_path"/>
   <arg name="ignore_set" default="[]" /> <!-- custom list of boxes not to draw in rviz -->
   <arg name="target_frame_dic" default="{'table_2':'table_3'}" /> <!-- express table_2 in table_3 reference frame -->
 
   <!-- rviz visualization of planning scene as markers, by default it publishes to tf -->
   <node pkg="grasplan" type="visualize_planning_scene_node" name="visualize_planning_scene_node" output="screen">
     <!-- planning scene transforms -->
-    <param name="yaml_path_to_read" type="string" value="$(arg yaml_path_to_read)" />
-    <!-- yaml file path to save boxes if requested-->
-    <param name="yaml_path_to_write" type="string" value="$(arg yaml_path_to_write)" />
+    <param name="yaml_path" type="string" value="$(arg yaml_path)" />
```

Comment:
This won't work, because the visualize_planning_scene_node doesn't read a `yaml_path` parameter. The full list of parameters is:

```
- ~/colors
- ~/ignore_all_but
- ~/ignore_set
- ~/target_frame_dic
- ~/transparency
- ~/yaml_path_to_read
- ~/yaml_path_to_write
```

So we have to keep the `yaml_path_to_read` and `yaml_path_to_write` parameters here.

Reply:
Funny, I tested it with providing the planning scene from the `/config` folder and it shouldn't have worked, however it's set as a default if the `yaml_path_to_read` and `yaml_path_to_write` parameters are not provided https://github.com/DFKI-NI/grasplan/blob/05d804cd4eee60fc010d109396deb8ea47b9c870/scripts/visualize_planning_scene_node#L19-L21

Reply:
Oooh, sneaky!